docs/pages/zkvm/sdk-benchmarking.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: "Nexus zkVM SDK Benchmark"
+lang: en-US
+description: "Benchmark your Nexus project using the SDK"
+image: "/nexus-head.png"
+---
+
+import Image from "next/image";
+
+import { Steps } from "nextra/components";
+
+## Reproducible Proving and Verification
+
+<Steps>
```

Comment:
Vercel is rendering three numbered steps: "RISC-V Cycles count" "Execution time" "Graphs". Did you want these three steps?

Reply:
Yup. I want to show the throughout benchmark has 3 parts, the storyline is:
1. At guest side: Code can be profiled base on the number of RISC-V cycles
2. At host side: Proving time can be profiled. 
3. The combine results of 1 and 2. 
What do you think?

Reply:
Those three steps make sense, so there will be headlines called something like
* guest side benchmark
* host side benchmark
* combining guest & host benchmarks